app/authentication/user_id_generator.py

```diff
@@ -0,0 +1,30 @@
+from app import settings
+from app.metadata.metadata_store import MetaDataConstants
+from app.utilities.strings import to_str
+import hashlib
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class UserIDGenerator(object):
+
+    @staticmethod
+    def generate_id(token):
```

Comment:
This could just be a method inside this file (i.e. outside of a class object), I suggest this would be a coding standard thing as to whether you always put things inside classes or not, as long as its consistent you can do either approach.